pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -151,41 +54,4 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 			},
 		}, 0),
 	)
-
-	It("should reject with JSON patch validation invalid", func() {
-		kvAdmitter := getAdmitter(true)
-		kv := getKV()
-		kvBytes, _ := json.Marshal(&kv)
-
-		cc := getComponentConfig()
-		kv.Spec.Workloads = &cc
-		kv.Spec.CustomizeComponents = v1.CustomizeComponents{
-			Patches: []v1.CustomizeComponentsPatch{
-				{
-					ResourceName: "virt-api",
-					ResourceType: "Deployment",
-					Type:         v1.StrategicMergePatchType,
-					Patch:        ``,
```

Comment:
It may be worth adding a table entry with `Patch: ""` to the table above.

Reply:
@rmohr Done.